apps/frontend/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,36 @@
+import { memo } from "react";
+import styles from "./Avatar.module.scss";
+import { logoColors } from "../../lib/utils/consts";
+
+type AvatarProps = { readonly img?: string; readonly name?: string };
+
+export const Avatar = memo<AvatarProps>(({ img, name }) => {
+  if (img) {
+    return (
+      <div className={styles.logo} style={{ backgroundImage: `url(${img})` }}>
+        <span className="sr-only">company logo</span>
+      </div>
+    );
+  }
+
+  if (name) {
+    const splittedName = name.split(" ");
+
+    return (
+      <div
+        className={styles.logo}
+        style={{
+          backgroundColor: `#${
+            logoColors[Math.floor(Math.random() * logoColors.length)]
```

Comment:
To się losuje przy każdym renderze, po co?

Reply:
To znaczy nie przy każdym, bo jest memo(), a jak można by to było inaczej zoptymalizować?